etc/parameters/default.json

```diff
@@ -1530,6 +1530,7 @@
     "look_at_dispatch_delay": {
       "nanos": 0,
       "secs": 1
-    }
+    },
+    "use_stand_head_unstiff_calibration": false
```

Comment:
please make true default for now until we make the switch to automatic calibration

Reply:
done